awx/main/utils/filters.py

```diff
@@ -175,6 +177,17 @@ def _json_path_to_contains(self, k, v):
                 v = self.strip_quotes_traditional_logic(v)
                 return (k, v)
 
+            for match in models.Field.get_lookups().keys():
```

Comment:
If people have legacy data _before_ this change like:

`ansible_facts__ansible_distribution__startswith=Cent`

...it will get truncated to:

`ansible_facts__ansible_distribution=Cent`

...and you'll see a loud warning in the logs.